lib/raygun/raygun.rb

```diff
@@ -105,6 +107,8 @@ def copy_prototype
 
       FileUtils.mv         dirs_to_move, app_dir
       FileUtils.remove_dir extraneous_dir
+
+      fetch_rubocop_file
```

Comment:
Could we guard this so we only do it for C5 repos? That way if raygun is used with a third-party template we are not potentially clobbering their rubocop config.

```suggestion
      fetch_rubocop_file if @prototype_repo == CARBONFIVE_REPO
```

Sort of like we do here: https://github.com/carbonfive/raygun/blob/03c07ebd8208e8f804d03668f2ca7b726b98cd86/lib/raygun/raygun.rb#L173-L179